focus_converter_base/focus_converter/conversion_configs/gcp/2_effective_cost_S002.yaml

```diff
@@ -0,0 +1,14 @@
+# sql query to compute effective_cost multiplied by usage.amount ONLY WHEN usage unit and pricing unit are the same. Otherwise show N/A
+plan_name: compute effective_cost
+conversion_type: sql_query
+column: tmp_credits_amount_EffectiveCost
+focus_column: EffectiveCost
+conversion_args: "SELECT
+    *,
+    CASE
+        WHEN PricingUnit = UsageUnit THEN tmp_effective_price_EffectiveCost * UsageQuantity
```

Comment:
I don't see a field named UsageUnit. Am i am missing something ?

Reply:
I was using UsageUnit from the previous step conversion. This way the plan is more efficient.
Similarly for PricingUnit and UsageQuantity.